extensions/machine-learning/src/views/widgets/dashboardWidget.ts

```diff
@@ -70,15 +64,15 @@ export class DashboardWidget {
 						position: 'absolute'
 					}).component();
 				mainContainer.addItem(container, {
-					CSSStyles: { 'padding-top': '25px', 'padding-left': '5px' }
+					CSSStyles: { 'padding-top': '12px' }
 				});
 				await view.initializeModel(mainContainer);
 				resolve();
 			});
 		});
 	}
 
-	private createHeader(view: azdata.ModelView): azdata.Component {
+	private async createHeader(view: azdata.ModelView): Promise<azdata.Component> {
```

Comment:
Why does this need to be async

Reply:
Header contains the tasksContainer which checks:
!await this._predictService.serverSupportOnnxModel()
I do not know what this check is for so I left it there and updated related methods to async.

Reply:
Oh right, I didn't see that. Makes sense!